src/Public/OpenApi.ps1

```diff
@@ -3217,10 +3238,10 @@ function Add-PodeOAExternalRoute {
             foreach ($tag in $DefinitionTag) {
                 #add the default OpenApi responses
                 if ( $PodeContext.Server.OpenAPI.Definitions[$tag].hiddenComponents.defaultResponses) {
-                    $extRoute.OpenApi.Responses = $PodeContext.Server.OpenAPI.Definitions[$tag].hiddenComponents.defaultResponses.Clone()
+                    $extRoute.OpenApi.Responses = Get-PodeOrderedDictionaryClone -Source $PodeContext.Server.OpenAPI.Definitions[$tag].hiddenComponents.defaultResponses
```

Comment:
`-Source` here should be `-Original` - though I do kind of prefer the "Source" parameter name over "Original"